.github/workflows/lint.yml

```diff
@@ -0,0 +1,36 @@
+name: Lint
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x]
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2.3.4
+
+      - name: Setup Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2.1.5
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Install dependencies with yarn
+        run: yarn
+
+      - name: Check types
+        run: yarn type-check
+
+      - name: Prettier
+        run: yarn prettier:check
```

Comment:
Please check if this (`yarn prettier:check` is the version of the prettier command you want to use here.